api/Notes/noteModel.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+
+const noteModel = {
+    title: {
+        type: String,
+        required: true,
+        unique: true
+    },
+
+    contents: {
+        type: String,
+        required: true
+    },
+
+    tags: {
+        type: Array,
+        default: []
+    },
+
+    timestamp: {
+        type: Date,
+        default: Date.now(),
+        required: true
+    }
```

Comment:
I don't see any fields in the Note or User models that links one to the other. How will you keep track of which notes belong to which user?